pyxnat/core/derivatives/ashs.py

```diff
@@ -0,0 +1,47 @@
+XNAT_RESOURCE_NAME = 'ASHS'
+
+def stats(self, mode='corr_nogray'):
```

Comment:
Function name is a bit vague compared to FreeSurfer's one (`hippoSfVolumes()`), perhaps in a nearby future with more derivative we might consider harmonizing them.

Reply:
I agree. I did not want to use `hippoSfVolumes` which is very FreeSurfer-connoted, and did not want either to add to the existing terminology. I should have used `volumes` instead of `stats` which refers to the name of the actual file generated by ASHS. What do you think?

Reply:
See 9dc19f45ba883.

Reply:
Excellent